tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -3768,10 +3778,6 @@ node_modules/chrome-devtools-frontend/front_end/components/Linkifier.js(504,31):
 node_modules/chrome-devtools-frontend/front_end/components/Linkifier.js(505,28): error TS2339: Property '_linkHandlerSettingInstance' does not exist on type 'typeof Linkifier'.
 node_modules/chrome-devtools-frontend/front_end/components/Linkifier.js(506,67): error TS2555: Expected at least 2 arguments, but got 1.
 node_modules/chrome-devtools-frontend/front_end/components/Linkifier.js(508,33): error TS2339: Property '_linkHandlerSettingInstance' does not exist on type 'typeof Linkifier'.
-node_modules/chrome-devtools-frontend/front_end/components/Linkifier.js(564,9): error TS2322: Type '({ section: string; title: any; handler: () => void; } | { section: string; title: string; handler: any; })[]' is not assignable to type '{ title: string; handler: () => any; }[]'.
```

Comment:
auto types should never have applied here, but I'm not sure why they no longer do.